### DISCUSSION TITLE
Synchronise RSS sync intervals with different *arrs

### DISCUSSION THREAD

**Discussion by mat926** [autobrr/autobrr#1801] [Ideas]
When I add an indexer with an RSS in autobrr and set the refresh interval, I can see in Prowlarr logs when it gets refreshed. The problem is the *arrs also have their own intervals and that gets run too in addition with the ones from autobrr. This could cause issues with the indexers due to concurrent requests being sent. 

In Autobrr I set up PrivateHD indexer as a Generic Torznab and gave it the Prowlarr link url. 
Here are the trace logs from Prowlarr showing an RSS request being sent for PrivateHD.  Notice this does not have any categories or season/episode information. This one is being sent from autobrr. 
![image](https://github.com/user-attachments/assets/cfc52032-5937-442d-ab43-b123f2ece357)
Here is another log entry for the same indexer within 5 minutes, this has categories and season/episode. This one is being sent by Sonarr. 
![image](https://github.com/user-attachments/assets/f8fe1216-b79e-49de-9830-0b565d764179)

Sonarr and Radarr setting each have different RSS intervals set. And currently there's no way to configure them per indexer. 
Cross-seed also has its own RSS intervals. 

Autobrr could communicate with the arr apps and check if there are conflicting RSS sync intervals and disable them or give a warning.  Have autobrr manage the RSS instead of the *arr apps.

**Comment by zxcvdr** [comment]
Adding on, it'd also be nice to be to group together RSS feeds within autobrr itself. Sometimes it is easier to create a separate feed for a specific item since most sites let you be very precise with your feed options. Being able to run all RSS feeds in a single chunk at the same time would help with concurrent request issues as well
